src/Features/MobileAppBanner.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Determine if the mobile app banner shows on Android devices
+ */
+class MobileAppBanner {
+	/**
+	 * Class instance.
+	 *
+	 * @var Analytics instance
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Hook into WooCommerce.
+	 */
+	public function __construct() {
+		add_filter( 'woocommerce_admin_get_user_data_fields', array( $this, 'add_user_data_fields' ) );
```

Comment:
A very thin feature class that just adds our user data field. This matches the patterns elsewhere and I couldn't think of a much better place to put it.. (unless it gets jammed in FeaturePlugin somewhere?) Keen for feedback on this.